unittests_bash/test_utils.bats

```diff
@@ -674,10 +674,17 @@ EOF
         }
     ]
 }
+{
+    "schema": "olm.bundle",
+    "name": "kubevirt-hyperconverged-operator.v4.17.5",
+    "package": "kubevirt-hyperconverged-v1",
+    "image": "registry.redhat.io/container-native-virtualization/hco-bundle-registry@sha256:12345",
```

Comment:
nit: I think it would be a good idea to use a sha of valid length here to better represent reality.

Reply:
fixed, thanks!